### ISSUE TITLE
Versioning of a care plan resource for a patient

### ISSUE THREAD

**Issue by joostholslag** [SanteonNL/shared-care-planning#36]
How do you handle versioning of a care plan/team resource instance? Version history should be kept for legal reasons. And changes should be visible to viewers.

**Comment by bramwesselo** [comment]
Yes, the CP-service should do that (reading version-history is a standard feature within FHIR). 
CP-contributors are notified as CarePlan/CareTeam/Tasks change; they are also free to keep/archive these instances.
